scanner/services/matcher.go

```diff
@@ -112,8 +110,19 @@ func (s *matcherService) parseIndexReport(contents *v4.Contents) (*claircore.Ind
 	return ir, nil
 }
 
-func (s *matcherService) GetMetadata(_ context.Context, _ *types.Empty) (*v4.Metadata, error) {
-	return nil, status.Error(codes.Unimplemented, "method GetMetadata not implemented")
+func (s *matcherService) GetMetadata(ctx context.Context, _ *types.Empty) (*v4.Metadata, error) {
+	lastVulnUpdate, err := s.matcher.GetLastVulnerabilityUpdate(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("getting last vulnerability update time: %w", err)
+	}
+
+	timestamp, err := types.TimestampProto(lastVulnUpdate)
+	if err != nil {
+		return nil, fmt.Errorf("internal error: %w", err)
+	}
+	return &v4.Metadata{
+		LastVulnerabilityUpdate: timestamp,
```

Comment:
TODO: if https://github.com/stackrox/stackrox/pull/9355 is merged first, be sure to set the scanner version